cull_duplicates.py

```diff
@@ -0,0 +1,6 @@
+import pandas as pd
+
+fp = 'data/pdb_data_seq.csv'
+df = pd.read_csv(fp)
+df = df.drop_duplicates(subset=['structureId'])
```

Comment:
Very nice, I approve the Pandas approach. Even if we are adding a dependency, it is probably a good one for the future.